pydoctor/test/__init__.py

```diff
@@ -25,3 +27,42 @@ class CapSys(Protocol):
         def readouterr(self) -> CaptureResult: ...
 else:
     CapLog = CaptureResult = CapSys = object
+
+
+class InMemoryWriter:
+    """
+    Minimal template writer that doesn't touches the filesystem.
```

Comment:
```suggestion
    Minimal template writer that doesn't touch the file system and doesn't record output.
```
This is useful to have. For other tests we might want to record the output into an in-memory buffer, but we can add that when we actually need it.

Reply:
For me this is useful as it shows the public interface of the writer :)

I think that most probably this will be extended to record the actions.... or output.

In the current format it is only epydoc2stan... I don't think that it will handle RST.